tests/sentry/integrations/msteams/test_message_builder.py

```diff
@@ -36,6 +45,35 @@ class MSTeamsMessageBuilderTest(TestCase):
     These tests do NOT test all visual aspects of the card.
     """
 
+    def setUp(self):
+        self.user = self.create_user(is_superuser=False)
+        owner = self.create_user()
+        self.org = self.create_organization(owner=owner)
+
+        self.integration = Integration.objects.create(
+            provider="msteams",
+            name="Fellowship of the Ring",
+            external_id="f3ll0wsh1p",
+            metadata={
+                "service_url": "https://smba.trafficmanager.net/amer",
+                "access_token": "y0u_5h4ll_n07_p455",
+                "expires_at": int(time.time()) + 86400,
+            },
+        )
+        OrganizationIntegration.objects.create(organization=self.org, integration=self.integration)
+
+        self.project1 = self.create_project(organization=self.org)
+        self.event1 = self.store_event(
+            data={"message": "oh no"},
+            project_id=self.project1.id,
+        )
+        self.group1 = self.event1.group
```

Comment:
This is a group of cards; right?

Reply:
Nope, this is a Sentry `group` which, if I understand correctly, is a group of issues.